charts/incubator/minecraft-bungeecord/questions.yaml

```diff
@@ -1,21 +1,6 @@
 # Include{groups}
 portals: {}
 questions:
-  - variable: portal
-    group: "Container Image"
-    label: "Configure Portal Button"
-    schema:
-      type: dict
-      hidden: true
-      attrs:
-        - variable: enabled
-          label: "Enable"
-          description: "enable the portal button"
-          schema:
-            hidden: true
-            editable: false
-            type: boolean
-            default: true
```

Comment:
It does not have a portal right?

Reply:
no even the other minecraft charts shouldnt have it.